src/nvim/msgpack_rpc/helpers.c

```diff
@@ -48,15 +47,17 @@ static msgpack_sbuffer sbuffer;
   } \
   \
   static void msgpack_rpc_from_##lt(Integer o, msgpack_packer *res) \
+  /* *INDENT-OFF* */ \
```

Comment:
this is hideously ugly, but imma gonna de-monomorphise this entire block very soon, so it doesn't matter either way if this is like the only case or two in the codebase.

Reply:
yeah, we should also use maybe a prefix to give more of a hint to contributors about WTF this is for. There are many linters and formatters and other things, so a prefix like `fmt/indent-off` or `uncrustify/indent-off` helps. Also if we ever switch to some other formatter, then these old noise markers are hard to find.

Reply:
Changed it to `uncrustify:indent-off` and `uncrustify:indent-on` :+1:

Reply:
I tested it on all other files in the repo, including the ones in `mpack` and `xdiff` and they all work fine. This isn't needed anywhere else.